diff --git a/include/msvcrt/corecrt_wstring.h b/include/msvcrt/corecrt_wstring.h
--- a/include/msvcrt/corecrt_wstring.h
+++ b/include/msvcrt/corecrt_wstring.h
@@ -70,7 +70,7 @@ _ACRTIMP wchar_t* __cdecl wcsspn(wchar_t*,const wchar_t*);
 _ACRTIMP wchar_t* __cdecl wcstok_s(wchar_t*,const wchar_t*,wchar_t**);
 _ACRTIMP size_t   __cdecl wcsxfrm(wchar_t*,const wchar_t*,size_t);

-#ifdef _UCRT
+#if _MSVCR_VER >= 140
 _ACRTIMP wchar_t* __cdecl wcstok(wchar_t*,const wchar_t*,wchar_t**);
 static inline wchar_t* _wcstok(wchar_t* str, const wchar_t *delim) { return wcstok(str, delim, NULL); }
 #  ifdef __cplusplus
